front-end/src/app/reports/submission-workflow/submit-report-step2.component.html

```diff
@@ -83,20 +85,20 @@ <h3>Enter password</h3>
               If you any questions on how to enroll, confirming your email, two-factor authentication, or your Personel
               Key please go to the following page:
               <a href="https://webforms.fec.gov/psa/help.htm" target="_blank" rel="noopener"
-              >FEC Electronic Filing Password Assignment System Help</a
+                >FEC Electronic Filing Password Assignment System Help</a
               >
             </p>
           </div>
         </div>
         <div class="col-6">
           <div class="field">
-            <label for="filing_password">
+            <label for="filingPassword">
               <strong>ELECTRONIC FILING PASSWORD </strong>
             </label>
-            <input type="text" pInputText id="filing_password" formControlName="filing_password"/>
+            <input type="text" pInputText id="filingPassword" formControlName="filingPassword" />
```

Comment:
Let's make this type="password" to hide it when being typed out.

Reply:
side effects of setting it to type="password" moved this out of scope